app/models/game.rb

```diff
@@ -61,4 +63,30 @@ def make_queens!
     Queen.create(game_id: self.id, color: 'white', x_cord: 3, y_cord: 0)
     Queen.create(game_id: self.id, color: 'black', x_cord: 3, y_cord: 7)
   end
+
+  def check?
+    black_king_in_check? || white_king_in_check?
+  end
+
+  def black_king_in_check?
```

Comment:
Just as an exercise: 

Could we merge the two methods (black_king_in_check? and white_king_in_check?) in one? They seem pretty identical..